app/workers/scheduler/user_cleanup_scheduler.rb

```diff
@@ -24,4 +25,10 @@ def clean_suspended_accounts!
       Admin::AccountDeletionWorker.perform_async(deletion_request.account_id)
     end
   end
+
+  def clean_discarded_statuses!
+    Status.discarded.where('deleted_at <= ?', 30.days.ago).find_each do |status|
+      RemovalWorker.perform_async(status.id, immediate: true)
+    end
+  end
```

Comment:
Naive question: is this efficient?

Reply:
Valid point, this is not at all efficient, looks like it's a seq scan.